cmd/pbapi/main.go

```diff
@@ -50,6 +51,15 @@ func main() {
 		log.Fatal().Err(err).Msg("Error initializing database")
 	}
 
+	// initialize the job queue
+	ctx := context.Background()
+	err = jobs.Initialize(ctx, &logger)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error initializing dejq queue")
+	}
+	jobs.RegisterJobs(&logger)
+	jobs.StartDequeueLoop(ctx, &logger)
```

Comment:
The comment above hopefully explains it - its a signal waiting pattern. The reason why this is done this way is that the moment the process receives INT or TERM signals we want to do a graceful shutdown. That's exactly what Queue.Stop() function does - it stops receiving of new jobs and it waits until all already started jobs are finished.

If we did not do that, the jobs would be killed immediately, that's not what we want to do.

Reply:
Oh right, good point. For production deployments, we want to use pbworker of course. But for development, the idea is to have the option of running in-memory dejq (no sql/sqs needed). In order to be able to do that, there also must be job queue in the pbapi. In production, this will be turned off.

In other words, if you want to test this patch, just enable "memory" setting and run pbapi to test the queue.